### ISSUE TITLE
an Organiser should only have to enter an amount to add an additional service (cost)

### ISSUE THREAD

**Issue by bo-bok** [chrishigglers/Higglers#30]
## why
Currently, an Organizer clicks to select the additional services they offer and then types in the charge for each service selected. We feel that there's a lot of unnecessary of clicking involved here.

![screen shot 2017-07-11 at 16 46 15](https://user-images.githubusercontent.com/19147734/28077058-80f376fe-6658-11e7-96a2-33b20eb78183.png)
[Link to inVision screen](https://projects.invisionapp.com/share/X3CIDTEMJ#/screens/242256765)

## what
remove the checkboxes from the additional costs section in **section 4 - payment** (for Organisers)

## how
* the price for each additional service would initially be displayed as zero (by default)
* If a user sets a new charge for a service, they are implicitly saying they want to add it (and thus a check box is not necessary)

**Comment by chrishigglers** [comment]
@bo-bok 
I disagree, I quite like the checkboxes I feel that they make the event organiser take a little bit more time over this section which is quite important as anything to do with cost has an impact to the end user. 

If there is a technical reason why we should remove these then happy to discuss.

_Query_
On the actual event profile I only want the additional costs to be displayed if they have been ticked. If no additional costs have been ticked then it should be blank (if possible - if not a compromise would be to grey the section out).
